src/types/venues.ts

```diff
@@ -434,6 +435,10 @@ export interface RecentSeatedUserTimestamp<T extends VenueTemplate>
   lastSittingTimeMs: number;
 }
 
+export interface SpacesDropdownItemList {
+  venueSpaces: { template?: PortalTemplate; name: string }[];
+}
+
```

Comment:
Have already commented elsewhere that this is unnecessary interface.

Reply:
removed da7ba06